src/blocks/recommendations/components/parsely-recommendations-list-item.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { Card, CardBody, CardMedia } from '@wordpress/components';
+
+const getImageForLink = ( { imagestyle, imageUrl, thumbUrlMedium } ) => {
+	if ( imagestyle === 'original' ) {
+		return imageUrl;
+	}
+	return thumbUrlMedium;
+};
+
+const ParselyRecommendationsListItem = ( {
+	imageAlt,
+	imagestyle,
+	recommendation: {
+		title: linkTitle,
+		url: linkUrl,
+		image_url: imageUrl,
+		thumb_url_medium: thumbUrlMedium,
+	},
+	showimages,
+} ) => {
+	const imageForLink = showimages && getImageForLink( { imagestyle, imageUrl, thumbUrlMedium } );
+
+	return (
+		<li>
+			<a href={ linkUrl } className="parsely-recommendations-link">
+				<Card className="parsely-recommendations-card" size="custom">
+					{ imageForLink && (
+						<CardMedia className="parsely-recommendations-cardmedia">
+							<img
+								className="parsely-recommendations-list-img"
```

Comment:
Why does the classname here include `-list` when the other classes (also in a list), don't?

Also, the element is already called `img`, so should the class name end with `-image` instead?